recipe/build.sh

```diff
@@ -23,47 +23,83 @@ if [[ "$CONDA_BUILD_CROSS_COMPILATION" == "1" ]]; then
   EXTRA_CONFIGURE_OPTIONS="--enable-cross-compile --arch=$ARCH --target-os=$OS --cross-prefix=$HOST- --host-cc=$CC_FOR_BUILD"
 fi
 
-if [[ "${target_platform}" == "linux-64" ]]; then
-  extra_args=--enable-vaapi
+extra_args=""
+if [[ "${target_platform}" == "win-64" ]]; then
+  extra_args="${extra_args} --target-os=mingw64"
```

Comment:
following doesn't work?
```suggestion
  extra_args="${extra_args} --target-os=win64"
```